Fix stage checking

a-ws-m opened the pull request:
Quick PR that solves #272 . I forgot I'd done this a while ago!

a-ws-m committed:
Fix stage checking

a-ws-m committed:
Merge branch 'develop' of https://github.com/a-ws-m/MDPOW into develop

orbeckst commented:
closing and opening to cycle CI

orbeckst committed:
Merge branch 'develop' into develop

orbeckst committed:
Merge branch 'develop' into develop

orbeckst reviewed (approved):
lgtm, thanks for the fix & sorry for the long delay!

orbeckst committed:
Merge branch 'develop' into develop

orbeckst committed:
Merge branch 'develop' into develop

orbeckst committed:
added tests and docs for mdpow.restart

- add tests for mdpow.restart
- clarified docs for Journal and Journalled
- update CHANGES
